deployments/k8s/helm/signalfx-agent/values.yaml

```diff
@@ -280,3 +280,8 @@ loadPerCPU: false
 # A list of monitor configurations to include in the agent config.  These
 # values correspond exactly to what goes under 'monitors' in the agent config.
 monitors:
+
+# For a complex monitor configurations, split into multiple files that will be
+# included into a configmap. These files are automatically included in the previous
+# monitors section if it used.
+monitorsd:
```

Comment:
Is this a boolean value or what?

Reply:
I added an example of usage of this parameter.